src/services/UiStakeDataService.ts

```diff
@@ -0,0 +1,76 @@
+import { UiStakeDataProvider } from '@aave/contract-helpers';
+import { Provider } from '@ethersproject/providers';
+import { Hashable } from 'src/utils/types';
+
+export type StakeGeneralUiData = {
```

Comment:
There's a type in the utilities package you can use for this, `GeneralStakeUIDataHumanized`

Reply:
I deleted the types since they were being exported for functionality but now since we have it on the library they are getting infered correctly